tests/webapp/api/test_jobs_api.py

```diff
@@ -168,61 +167,64 @@ def test_job_list_in_filter(webapp, eleven_jobs_stored, test_repository):
                        "f1c75261017c7c5ce3000931dce4c442fe0a1297,"
                        "9abb6f7d54a49d763c584926377f09835c5e1a32")
 
-    resp = webapp.get(final_url).json
-    assert len(resp['results']) == 2
+    resp = client.get(final_url)
+    assert resp.status_code == 200
+    assert len(resp.json()['results']) == 2
 
 
-def test_job_detail(webapp, test_job):
+def test_job_detail(client, test_job):
     """
     test retrieving a single job from the jobs-detail
     endpoint.
     """
-    resp = webapp.get(
+    resp = client.get(
         reverse("jobs-detail",
                 kwargs={"project": test_job.repository.name,
                         "pk": test_job.id})
     )
-    assert resp.status_int == 200
-    assert isinstance(resp.json, dict)
-    assert resp.json["id"] == test_job.id
-    assert not resp.json.get("taskcluster_metadata")
+    assert resp.status_code == 200
+    assert isinstance(resp.json(), dict)
```

Comment:
I don't know how expensive the call to ``resp.json()`` is, but I'd be tempted to set that to a temp var and then act on that.  This is such a nit that it really doesn't matter much, just something that jumped out at me.  :)  Feel free to ignore, if you like.  :)

Reply:
> I don't know how expensive the call to resp.json() is

Ah same thing occurred to me - but I checked it and turns out it's memoized:
https://github.com/django/django/blob/1.11.12/django/test/client.py#L693-L701